activestorage/app/models/active_storage/blob.rb

```diff
@@ -153,7 +174,7 @@ def signed_id(purpose: :blob_id)
   # Always refer to blobs using the signed_id or a verified form of the key.
   def key
     # We can't wait until the record is first saved to have a key for it
-    self[:key] ||= self.class.generate_unique_secure_token(length: MINIMUM_TOKEN_LENGTH)
+    self[:key] ||= self.class.generate_unique_interpolated_secure_key(length: MINIMUM_TOKEN_LENGTH)
```

Comment:
Hmm, there’s no interpolation going on here. This also won’t work since `key`, `record` and `attachable` are required kwargs of `generate_unique_interpolated_secure_key`.

Reply:
Indeed this is not a required change.